Frontend/ui-customer/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -348,6 +348,15 @@ config =
             { radius : 50000
             , strictBounds : true
             }
+        , enableCabs : false
```

Comment:
@MercyQueen `enableCabs` For what we are using this config?

Reply:
This is for changing the illustration which would be visible on home screen when there is no suggested trip or recent searches.